_layouts/resource.html

```diff
@@ -4,7 +4,6 @@
 ---
 
 <div class="page-header">
-  <a class="btn btn-ghost pull-right" href="{{ page.url | regex_replace: '\.html$', '-console.html' | prepend: site.baseurl }}">API console</a>
   <h1>{{ page.title }}</h1>
```

Comment:
Now we do not need a customized title any longer, so you can drop the `<div class="page-header"/>` completely and remove the `title_hidden: true` from the frontmatter.